person.rb

```diff
@@ -0,0 +1,25 @@
+# person.rb
+
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- Hey, it is good to note that the shouldn't be passed in the parameters as you've done here because they are only meant to be retrieved as you've added in the setter method, therefore, you should remove it out from the constructor, and generate the id automatically to ensure it's unique. You can achieve that by doing 
`@id = Random.rand(1..1000)`. Please kindly make this change.